Tests/private_build/upload_packs_private.py

```diff
@@ -227,19 +227,30 @@ def create_and_upload_marketplace_pack(upload_config: Any, pack: Any, storage_bu
         pack.cleanup()
         return
 
+    if not is_private_build:
```

Comment:
@guykeller when will this be true or not?(see that it is not a new code, just moved up)
why we don't run `pack.detect_modified` if `is_private_build` is true?